### ISSUE TITLE
Style of JavaDoc is Broken

### ISSUE THREAD

**Issue by marchof** [jacoco/jacoco#110]
Since release 0.6.3 the formatting of the JavaDoc shipped with the documentation is broken.

**Comment by marchof** [comment]
@Godin As we have Java 5 now on ClaudBees: Can we close this issue?

**Comment by Godin** [comment]
@marchof no, anyway we must update POMs to forbid releases without Java 5.
